lib/python/rose/metadata_check.py

```diff
@@ -102,9 +102,8 @@ def _check_macro(value, module_files=None, meta_dir=None):
         return
     try:
         macros = rose.variable.array_split(value, only_this_delim=",")
-    except Exception as e:
-        return INVALID_SYNTAX.format(e)
-    bad_macros = []
+    except Exception as exc:
```

Comment:
(You are right on this one - as the main failure is with a method that does a lot of regular expression matchings, but I think I'll give it a miss for now, in case the change breaks something else.)